Makefile

```diff
@@ -9,6 +9,11 @@ NON_MATCHING ?= 0
 # If ORIG_COMPILER is 1, compile with QEMU_IRIX and the original compiler
 ORIG_COMPILER ?= 0
 
+# Set prefix to mips binutils binaries (mips-linux-gnu-ld => 'mips-linux-gnu') - Change at your own risk!
+# In nearly all cases, not having 'mips-linux-gnu-*' binaries on the PATH is indicative of missing dependencies
+# This project is not maintained or tested on platforms that require changing this (like gentoo)
```

Comment:
I'm personally not sure about this last line, since we would probably be okay (?) with light changes to improve compatibility,
and I wouldn't say it's tested with `mips-linux-gnu` anyway apart from "the rom matches"

I think the "Change at your own risk!" above is enough to cover this